modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -221,7 +221,7 @@ The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used toge
 Neo4j has built-in support for a seed from a mounted file system (file), FTP server, HTTP/HTTPS server, Amazon S3, Google Cloud Storage, and Azure Cloud Storage.
 To select a seed provider, you need to configure xref:configuration/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[dbms.databases.seed_from_uri_providers].
 
-For more information on seeding from URI, refer to xref::clustering/databases.adoc#cluster-seed-uri[Manage databases in a cluster -> Seed from URI].
+For more details and examples of seeding from URI, refer to xref::clustering/databases.adoc#cluster-seed-uri[Manage databases in a cluster -> Seed from URI].
```

Comment:
My suggestion was to move it to the end of the page where the simple example is.

Reply:
I'll move it to the in another commit